### PR TITLE
convert most hard-wrapping to soft-wrapping

### PR DESCRIPTION
**Pull request by ljharb**
Per https://openjs-foundation.slack.com/archives/CS23BMK1P/p1687896210057419

This is much more readable when rendered, because not everyone views content on the same size screen (phones, in particular).

It's also much more readable when editing, which also causes less diff churn when making changes.

Recommended to view without whitespace changes. I made a few editorial corrections as well.

### CODE REVIEW

File: collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md [hunk 1]
```diff
@@ -90,7 +86,7 @@ This is an informational checklist to help start up a collaboration space into t
 - [ ] Adopt the OpenJS Foundation Code of Conduct
 - [ ] If choosing to use a Contributor License Agreement (CLA) or Developer Certificate of Origin (DCO), make selection and implement appropriate tool
 - [ ] Confirm required files in place (CODE_OF_CONDUCT.md, LICENSE.md, Overview.md, Governance.md) and approved by the CPC.
-- [ ] Publish Collobration Space Overview on Foundation Website
+- [ ] Publish Collabration Space Overview on Foundation Website
```
Comment (tobie) [inline collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md:89]:
```suggestion
- [ ] Publish Collaboration Space Overview on Foundation Website
```

**Review by tobie — APPROVED** [review]
On a cursory glance, this LGTM. Thanks for doing the work.

A little worried about the PR churn though, e.g. on #1055.

File: collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md (hunk 1 above)
Comment (ljharb) [reply collaboration-spaces/COLLABORATION_SPACE_PROGRESSION.md:89]: lol whoops, thanks

**Comment by ljharb** [comment]
re churn, once I land this, I'll go through every open PR and rebase them myself, to make sure nobody has to deal with any conflicts.

**Review by darcyclarke — APPROVED** [review]

**Review by Ethan-Arrowood — APPROVED** [review]

**Pull request merged**